ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите 1 катет: ");
+            double a = int.Parse(Console.ReadLine());
```

Comment:
Неверный парсер. В вашем варианте если ввести вс консоли не целое число, то дробная часть будет отброшена
```suggestion
            double a = double.Parse(Console.ReadLine());
```